bin/reformat-files.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Find all JS and JSX files in the project that have the @format tag
+ * and reformat them with Prettier. Useful when upgrading Prettier to
+ * a newer version.
+ * @format
+ */
+const fs = require( 'fs' );
+const glob = require( 'glob' );
+const ignore = require( 'ignore' );
+const path = require( 'path' );
+const prettier = require( 'prettier' );
+
+/**
+ * Returns true if the given text contains @format.
+ * within its first docblock. False otherwise.
+ *
+ * @param {String} text text to scan for the format keyword within the first docblock
+ */
+const shouldFormat = text => {
```

Comment:
It would be nice to share the `shouldFormat` here:

https://github.com/Automattic/wp-calypso/blob/641207b6f599e9c4e4a11b89821248467b204f4c/bin/pre-commit-hook.js#L31-L46